patches/api/0213-Add-additional-open-container-api-to-HumanEntity.patch

```diff
@@ -3,16 +3,134 @@ From: JRoy <joshroy126@gmail.com>
++     *     opened.
++     */
++    @Nullable
++    public InventoryView openWorkbench(@Nullable Location location, boolean force, @Nullable net.kyori.adventure.text.Component title);
++    // Paper end
++
++    /**
++     * Opens an empty enchanting inventory window with the player's inventory
++     * on the bottom.
++     *
++     * @param location The location to attach it to. If null, the player's
++     *     location is used.
++     * @param force If false, and there is no enchanting table at the
++     *     location, no inventory will be opened and null will be returned.
++     * @return The newly opened inventory view, or null if it could not be
++     *     opened.
++     */
++    @Nullable
++    // Paper start - overridable title
++    public default InventoryView openEnchanting(@Nullable Location location, boolean force) {
++        return openEnchanting(location, force, null);
++    }
+ 
+     /**
+      * Opens an empty enchanting inventory window with the player's inventory
+@@ -133,11 +169,13 @@ public interface HumanEntity extends LivingEntity, AnimalTamer, InventoryHolder
+      *     location is used.
+      * @param force If false, and there is no enchanting table at the
+      *     location, no inventory will be opened and null will be returned.
++     * @param title The title to use instead of the default one
+      * @return The newly opened inventory view, or null if it could not be
+      *     opened.
+      */
+     @Nullable
+-    public InventoryView openEnchanting(@Nullable Location location, boolean force);
++    public InventoryView openEnchanting(@Nullable Location location, boolean force, @Nullable net.kyori.adventure.text.Component title);
++    // Paper end
+ 
+     /**
+      * Opens an inventory window to the specified inventory view.
+@@ -158,7 +196,43 @@ public interface HumanEntity extends LivingEntity, AnimalTamer, InventoryHolder
+      * opened.
+      */
      @Nullable
-     public InventoryView openMerchant(@NotNull Merchant merchant, boolean force);
+-    public InventoryView openMerchant(@NotNull Villager trader, boolean force);
++    // Paper start - overridable title
++    public default InventoryView openMerchant(@NotNull Villager trader, boolean force) {
++        return openMerchant(trader, force, null);
++    }
++
++    /**
++     * Starts a trade between the player and the villager.
++     *
++     * Note that only one player may trade with a villager at once. You must use
++     * the force parameter for this.
++     *
++     * @param trader The merchant to trade with. Cannot be null.
++     * @param force whether to force the trade even if another player is trading
++     * @param name The name to use instead of the default one
++     * @return The newly opened inventory view, or null if it could not be
++     * opened.
++     */
++    @Nullable
++    public InventoryView openMerchant(@NotNull Villager trader, boolean force, @Nullable net.kyori.adventure.text.Component name);
```

Comment:
Use `title` to maintain consistency.

Reply:
My rationale for calling this parameter name instead of title was that when you set the title override to "Test" you would see "Test - Novice" instead of just "Test".

I think it is better to add some additional JavaDoc explaining that.

Should I still rename the param?